cmd/kubelet/app/options/options.go

```diff
@@ -397,6 +397,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Var(cliflag.NewColonSeparatedMultimapStringString(&c.StaticPodURLHeader), "manifest-url-header", "Comma-separated list of HTTP headers to use when accessing the url provided to --manifest-url. Multiple headers with the same name will be added in the same order provided. This flag can be repeatedly invoked. For example: --manifest-url-header 'a:hello,b:again,c:world' --manifest-url-header 'b:beautiful'")
 	fs.Var(&utilflag.IPVar{Val: &c.Address}, "address", "The IP address for the Kubelet to serve on (set to '0.0.0.0' or '::' for listening in all interfaces and IP families)")
 	fs.Int32Var(&c.Port, "port", c.Port, "The port for the Kubelet to serve on.")
+	fs.Var(&utilflag.IPVar{Val: &c.ReadOnlyBindAddress}, "read-only-bind-address", "The address used for the read-only API on, when enabled by setting --read-only-port. Defaults to the value passed to --address. Has no effect if ReadOnlyPort is disabled. (set to '0.0.0.0' or '::' for listening in all interfaces and IP families)")
```

Comment:
nits on text:

1. This sentence doesn't seem complete: "The address used for the read-only API on, when"
2. "Has no effect if ReadOnlyPort is disabled." - maybe use separate words instead of  ReadOnlyPort in pascal case. Alternatively use `--read-only-port` as a argument name.
3. Does '0.0.0.0' will set it to listen for IPv6 as well? I just don't know

Reply:
I see the text was updated. Can you confirm that setting '0.0.0.0' will set it to listen for IPv6 as well for my education.